SU-SWS/caw_profile
# CAW21-38

# READY FOR REVIEW

# Summary
- Convert from react paragraphs to layout paragraphs.

# Need Review By (Date)
- Aug 30th EOD

# Urgency
- high

# Steps to Test
using `caw` branch of the `ace-gryphon` stack
1. `composer require su-sws/caw_profile:dev-CAW21-38 --no-update`
2. `composer update -n`
3. `blt drupal:sync --site=cardinalatwork`
4. View your local and compare a few pages with the `https://cardinalatwork-prod.stanford.edu` site
5. Create a page and test out a few combinations of the layouts/paragraphs
6. ensure nothing critical breaks or looks terribly wrong. (styles on the admin side might not be perfect, but as long as its close enough)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
